package.json

```diff
@@ -10,6 +10,10 @@
         "backend/node_modules"
     ],
     "dependencies": {
-        "firebase-admin": "^10.0.0"
+        "chai-http": "^4.3.0",
+        "firebase-admin": "^10.0.0",
+        "install": "^0.13.0",
+        "mocha": "^9.1.3",
+        "npm": "^8.2.0"
```

Comment:
If these are only used on the back end, move them there to that `package.json`.
Also, why is NPM listed as a dependency?

Reply:
I removed it